build-ps/debian/rules_zenfs

```diff
@@ -0,0 +1,218 @@
+	@echo "RULES.$@"
+	
+	[ -f $(EXPORTED_SOURCE_TARBALL) ] || tar -zcf $(EXPORTED_SOURCE_TARBALL) \
+	--exclude=debian . \
+	--transform="s,^\./,percona-server/,"
+
+ifeq ($(SKIP_DEBUG_BINARY),)
+	cd $(builddebug) && $(MAKE) -j$(NCPU)
+endif
+
+	cd $(builddir) && $(MAKE) -j$(NCPU) VERBOSE=1
+
+	touch $@
+
+	cd $(builddir); \
+	cd rocksdb-source; \
+	CC=clang-12 CXX=clang++-12 make -j$(NCPU) DISABLE_WARNING_AS_ERROR=1 PREFIX=$(CURDIR)/rocksdb-root/usr OBJ_DIR=$(CURDIR)/rocksdb-build ROCKSDB_PLUGINS=zenfs install-static; \
+	cd plugin/zenfs/util; \
+	PKG_CONFIG_PATH=$(CURDIR)/rocksdb-root/usr/lib/pkgconfig CC=clang-12 CXX=clang++-12 make -j$(NCPU) 
+
+override_dh_auto_install:
+	@echo "RULES.$@"
+
+	# complete install first
+	(cd $(builddebug) && $(MAKE) install DESTDIR=$(TMPD)/)
+	(cd $(builddir) && $(MAKE) install DESTDIR=$(TMP)/)
+
+	# add Percona Server configuration file my.cnf to mysql-common package
+	#install -g root -o root -m 0644 -D debian/extra/my.cnf $(TMP)/etc/mysql/my.cnf
+
+	# add MySQL Server debug binary and library to package
+	#install -g root -o root -m 0755 debian/extra/server-binary $(TMP)/usr/sbin/mysqld-debug
+
+	# add systemd script
+	install -m 0755 debian/extra/mysql-systemd-start $(TMP)/usr/share/mysql/
+	install -m 0755 debian/extra/mysqlrouter-systemd-start $(TMP)usr/share/mysqlrouter/
+
+	# install default file for init script timeout params
+	install -d $(TMP)/etc/default
+	install -m 0644 debian/extra/default-mysql $(TMP)/etc/default/mysql
+        # Add helper functions for maintainer scripts
+	install -m 0644 debian/extra/mysql-helpers debian/tmp/usr/share/mysql/
+	# add apparmor profile
+#	install -g root -o root -m 0644 -D debian/extra/apparmor-profile $(TMP)/etc/apparmor.d/usr.sbin.mysqld
+#	install -g root -o root -m 0644 -D debian/extra/apparmor-profile-router $(TMP)/etc/apparmor.d/usr.bin.mysqlrouter
+
+
+	# install source tarball for source package
+	install -D -m 0644 $(EXPORTED_SOURCE_TARBALL) $(TMP)/usr/src/percona-server/`basename $(EXPORTED_SOURCE_TARBALL)`
+
+	# install AppArmor profile for Percona Server
+	install -d $(TMP)/etc/apparmor.d
+	install -d $(TMP)/etc/apparmor.d/local
+
+	install -m 0755 $(builddir)/rocksdb-source/plugin/zenfs/util/zenfs $(TMP)/usr/bin/zenfs
+
+override_dh_strip:
+	dh_strip --dbg-package=percona-server-dbg
+	dh_strip -Xlibprotobuf-lite
+
+override_dh_installinit:
+	@echo "RULES.$@"
+#	dh_apparmor -ppercona-server-server --profile-name=usr.sbin.mysqld
+#	dh_apparmor -ppercona-mysql-router --profile-name=usr.bin.mysqlrouter
+	if [ "$(DISTRELEASE)" != "precise" -a "$(DISTRELEASE)" != "trusty" -a "$(DISTRELEASE)" != "wheezy" ]; then dh_systemd_enable --name=mysql; dh_systemd_enable --no-enable --name=mysql@; fi
+	dh_installinit --name=mysql -- defaults 19 21
+	dh_installinit --name=mysqlrouter -- defaults 19 21
+	if [ "$(DISTRELEASE)" != "precise" -a "$(DISTRELEASE)" != "trusty" -a "$(DISTRELEASE)" != "wheezy" ]; then dh_systemd_start --restart-after-upgrade; fi
```

Comment:
as all supported Ubuntu now works with systemd we can modify this line and remove if statement

Reply:
Done